scripts/genwallet.py

```diff
@@ -1,33 +1,50 @@
 #!/usr/bin/env python3
 
-import sys
+# A script for noninteractively creating wallets.
+# The implementation is similar to wallet_generate_recover_bip39 in jmclient/wallet_utils.py
+
 import os
 from optparse import OptionParser
-from jmclient import load_program_config, add_base_options, SegwitWallet, SegwitLegacyWallet, create_wallet, jm_single
+from pathlib import Path
+from jmclient import (
+    load_program_config, add_base_options, SegwitWalletFidelityBonds, SegwitLegacyWallet,
+    create_wallet, jm_single, wallet_utils
+)
 from jmbase.support import get_log, jmprint
 
 log = get_log()
 
 def main():
     parser = OptionParser(
-    usage='usage: %prog [options] wallet_file_name password',
-    description='Create a wallet with the given wallet name and password.')
+        usage='usage: %prog [options] wallet_file_name [password]',
+        description='Create a wallet with the given wallet name and password.'
+    )
     add_base_options(parser)
+    parser.add_option(
+        '--recovery-seed-file',
+        dest='seed_file',
+        default=None,
+        help='File containing a mnemonic recovery phrase.'
```

Comment:
Maybe this help text could be improved to clearly indicate new wallet will be recovered from seed phrase in file (instead of generating new wallet from scratch and saving seed phrase to file)?

Reply:
Done, thanks!